src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
+
+export const ElMenu = styled.div`
+  overflow: hidden;
+  display: flex;
+  min-width: 200px;
+  width: fit-content;
+  padding: var(--spacing-2) var(--spacing-none);
+  border-radius: var(--corner-default);
+  background-color: var(--fill-white);
+  box-shadow: 0px 4px 16px 0px #222b3329;
+`
+
+export const ElMenuItemContainer = styled.li`
```

Comment:
**note:** One thing that isn't clear to me from the Design spec is whether menu items that do something asynchronously when clicked need to have a busy state, or if those kinds of async actions from menu items will be communicated in another manner.

Reply:
yeah, I didn't see any busy state styling in Design spec either so I assume if the list item clicked and its button, then it will close the menu (this logic handled in `Popover`)

Reply:
I think also because the menu typically used for navbar, it less necessary to provide a busy state 🤔